3.processing_features/scripts/0.merge_sc_cytotable.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Merge single cells from CellProfiler outputs using CytoTable
+
+# ## Import libraries
+
+# In[1]:
+
+
+# Modified Jenna's nf1_ic.ipynb file from the Cellpainting repo
+# https://github.com/WayScience/nf1_cellpainting_data/blob/main/3.processing_features/0.merge_sc_cytotable.ipynb
+
+import sys
+import pathlib
+import yaml
+import pprint
+import pandas as pd
+
+# cytotable will merge objects from SQLite file into single cells and save as parquet file
+from cytotable import convert, presets
```

Comment:
consider changing the import order: [here](https://peps.python.org/pep-0008/#imports)

Reply:
I believe I changed it to the correct order now. If you agree, I will modify all of my other scripts as well